For Host only, set disconnect/disband to disband only

taywrobel opened the pull request:
Change the functionality of the disconnect/diaband
button in the network fragment to serve as disband
at all times when you are host, and disconnect at
all times when you are guest.

Fixes #184
Fixes #183 by changing disband confirmation wording

taywrobel committed:
For Host only, set disconnect/disband to disband only

Change the functionality of the disconnect/diaband
button in the network fragment to serve as disband
at all times when you are host, and disconnect at
all times when you are guest.

dgreenhalgh commented:
@theJenix

theJenix commented on SoundStream/src/com/lastcrusade/soundstream/components/NetworkFragment.java:
```diff
@@ -150,17 +149,15 @@ public void onClick(View v) {
+		// this case, it should be a disband function, otherwise (when we are
+		// host) it should be be a disconnect function.
+		if (getConnectionService() == null
+				|| getConnectionService().isHostConnected()) {
```
Please fix the spacing here...the indentation is out of whack.

theJenix commented:
This looks good to me after you fix the :space_invader:  in NetworkFragment.java

taywrobel committed:
Standardized whitespace in NetworkFragment

Accidentally autoformatted with tabs.  Changed
eclipse to use 4 spaces from now on instead.

theJenix commented:
Looks :nine: to me.

dgreenhalgh commented:
:shipit:
